content/en/tracing/trace_collection/library_config/php.md

```diff
@@ -422,6 +425,28 @@ Note that `DD_TRACE_RESOURCE_URI_MAPPING_INCOMING` applies to only incoming requ
 When [`open_basedir`][9] setting is used, then `/opt/datadog-php` should be added to the list of allowed directories.
 When the application runs in a docker container, the path `/proc/self` should also be added to the list of allowed directories.
 
+### Headers extraction and injection
+
+The Datadog APM Tracer supports [B3][7] and [W3C][10] headers extraction and injection for distributed tracing.
+
+You can configure injection and extraction styles for distributed headers.
+
+The .NET Tracer supports the following styles:
+
+- Datadog: `Datadog`
+- W3C: `tracecontext`
+- B3 Multi Header: `b3multi` (`B3` is deprecated)
+- B3 Single Header: `B3 single header`
+
+You can use the following environment variables to configure injection and extraction styles. For instance:
+
+- `DD_TRACE_PROPAGATION_STYLE_INJECT=Datadog,tracecontext`
+- `DD_TRACE_PROPAGATION_STYLE_EXTRACT=Datadog,tracecontext`
+
+The environment variable values are comma-separated lists of header styles enabled for injection or extraction. By default, only the `Datadog` injection style is enabled.
+
+If multiple extraction styles are enabled, the extraction attempt is completed in order of configured styles, and uses the first successful extracted value.
```

Comment:
That's wrong, it's hardcoded in code. (tracecontext has prio, then datadog, then b3).

Reply:
Hmm, wait. The RFC says:

`When a service-entry span is created, headers from the incoming request are inspected for the specified distributed tracing styles in the order that they were provided. This applies to HTTP requests and, if applicable instrumentation exists for a tracing library, gRPC requests.`

Am I missing something?